mmv1/third_party/terraform/resources/resource_logging_sink.go

```diff
@@ -45,7 +45,7 @@ func resourceLoggingSinkSchema() map[string]*schema.Schema {
 		"exclusions": {
 			Type:        schema.TypeList,
 			Optional:    true,
-			Description: `Log entries that match any of the exclusion filters will not be exported. If a log entry is matched by both filter and one of exclusion_filters it will not be exported.`,
+			Description: `Log entries that match any of the exclusion filters will not be exported. If a log entry is matched by both filter and one of exclusion's filters, it will not be exported.`,
```

Comment:
```suggestion
			Description: `Log entries that match any of the exclusion filters will not be exported. If a log entry is matched by both filter and one of exclusions.filter, it will not be exported.`,
```
Sorry I'm not super familiar with this resource.  Do you think if it makes sense to change `exclusion_filters` to `exclusions.filter`? I assume the `exclusion_filters` on the [API doc](https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.sinks) just means the subfield `filter` of `exclusions`. Please correct me if I'm wrong.

Reply:
@shuyama1 You are totally right, I'll update this accordingly :+1: